### DISCUSSION TITLE
Mail server creation fails w/ error 500

### DISCUSSION THREAD

**Discussion by luismanson** [postalserver/postal#1620] [Help with using Postal]
Hello, im having issues with Postal failing to create the first mailserver. I just see error 500 comming from /org/testing/servers/new
I see no logs in journalctl nor docker.

Any help is appreciated.

**Comment by willpower232** [comment]
You should see logs with the command `postal logs web`

**Reply by luismanson** [reply]
there are no useful logs :\ running 'docker logs --follow postal_web_1'  only shows the post error, with no backtrace or something else. 
200.123.140.66 - - [09/Oct/2021:19:14:47 +0000] "POST /500 HTTP/1.1" 500 46 0.1310

**Reply by willpower232** [reply]
Ah thats interesting, @adamcooke would the rails logs be in the container somewhere or should they be output via docker too?

**Reply by tricelex** [reply]
@luismanson I had a similar issues days back, the logs only showed minimal information and not the source of the 500 error.
What I did was to cd into the running container then find the log of the running service in its installation directory.

**Reply by luismanson** [reply]
> @luismanson I had a similar issues days back, the logs only showed minimal information and not the source of the 500 error. What I did was to cd into the running container then find the log of the running service in its installation directory.

Thanks! 
I finally found this: 
`Mysql2::Error::ConnectionError` (Access denied for user 'postaluser'@'%' to database 'postal-server-8'):

This should be easy to fix..!

**Reply by Wouter0100** [reply]
Yes, @willpower232 - the rails logs are not yet send to stdout as far as I see can see. I've had this problem a couple of times as wel and had to look inside the containers themselves.
